common/lib/dataset.py

```diff
@@ -149,7 +153,8 @@ def __init__(self, parameters=None, key=None, job=None, data=None, db=None, pare
 				"software_file": "",
 				"num_rows": 0,
 				"progress": 0.0,
-				"key_parent": parent
+				"key_parent": parent,
+				"annotation_fields": json.dumps({})
 			}
 			self.parameters = parameters
```

Comment:
Probably want to define `self.annotation_fields` below here instead of above because else the property is never set when creating a new dataset